### ISSUE TITLE
How long for forwarding?(224x224 image)  

### ISSUE THREAD

**Issue by Bo331429856** [suzhenghang/MobileNetv2#3]
Hi, can you share your time for forwarding of a 224x224 image? And is it as fast as the speed reported in the mobilev2 paper.
      I have train this model with caffe, but the speed is lower than mobilenetv1, also the paper.

**Comment by suzhenghang** [comment]
@Bo331429856   yes, the speed is depended on the efficiency of the depthwise convolution layer.  The version of tensorflow Google used may do a lot of optimizations to speed up.

**Comment by AlexofNTHU** [comment]
HI @suzhenghang :
How is your performance on your machine?
Would you please report your FPS ?
